_syllabus/seed.py

```diff
@@ -76,12 +77,19 @@
         hwDate  = lecDate.replace(hours=15)
         solDate = hwDate.shift(weeks=1)
 
+
+	 
+	oh_plus_plus_link = wk{0:02d}-ohpp.md'.format(idx + 1)
+
```

Comment:
@mterwill What do you think about adding this to the seed file? If we run this next semester, travis will yell about a 404 if we forget to add the notes. Additionally, we don't always have OH++ notes for a lecture. Do we want to just add them manually as we go along in the semester?